app/api/chat/route.ts

```diff
@@ -10,9 +10,8 @@ export const POST = enhanceRouteHandler(
   async ({ body }) => {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const service = createChatLLMService();
-    const referenceId = "if1Fg9bo"; // hard coded for demo
-
-    try {    
+    const referenceId = body.accountId || "if1Fg9bo"; 
```

Comment:
Do we still need the fallback value?
- What happens if we `remove` the fallback value?